### ISSUE TITLE
Install instructions method 2 in readme.md result in build error "Unsupported class file major version 64"

### ISSUE THREAD

**Issue by Frenoss** [BrightSpots/rcv#702]
Hi, I expect I'm trying RCTab at the perfect time, for good and ill. Support for CSV looks to be added to the develop branch, and I'm betting it's just the alpha-version changes that are causing this problem for me.

My hope is to use RCTab to run about 20 RCV races, with 15-20 voters participating, in our local organization's voting in 10 days. The data are coming from Google Forms to Google Sheets. If I understand the features of RCTab, I could either find a way to export the sheet to the cast vote record CDF and tabulate it with the released 1.3.1, or I can save each race as a simple CSV and tabulate with the develop branch. The second seems like it would take much less time. That's my motivation for opening this issue.

**Comment by tarheel** [comment]
@artoonie @HEdingfield any idea what this would be about?

**Comment by Frenoss** [comment]
Sorry for not pasting this earlier. Here's more of the output.

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not open settings generic class cache for settings file 'C:\Code\rctab\settings.gradle' (C:\Users\frenoss\.gradle\caches\7.5.1\scripts\2927ee32z33hk0nq7er33ro2b).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 64
```

**Comment by artoonie** [comment]
You're probably using a newer version of Java -- try using 17 instead of 20. https://stackoverflow.com/questions/75878942/spring-initializr-project-results-in-unsupported-class-file-major-version-64

You might also consider not building from scratch, and instead using a prepackaged binary. What OS are you using?

**Comment by Frenoss** [comment]
Thank you, Java 17.0.2 builds fine.

For OS I'm using Windows 10 22H2. Am I right in understanding that the only prepackaged binaries are from the releases (1.3.1, 1.3.0, etc.)?

**Comment by artoonie** [comment]
Yes -- though we now have the ability to quickly build you a prerelease of develop, if you desire it. It sounds like you're good to go, so we'll hold off on doing that since it's a new ability and we haven't ironed out the process yet. Can we call this closed?

**Comment by Frenoss** [comment]
Thank you for all of the help.
